### ISSUE TITLE
Please add support online multilingual voices in filter

### ISSUE THREAD

**Issue by 1001ruchka** [gexgd0419/NaturalVoiceSAPIAdapter#32]
Microsoft recently added 3 online multilingual voices:
it-IT-GiuseppeMultilingualNeural
ko-KR-HyunsuMultilingualNeural
pt-BR-ThalitaMultilingualNeural
To display them, you have to enable all languages in the NaturalVoiceSAPIAdapter.
It would be great if multilingual voices were displayed at all filter values.

<img width="392" alt="image" src="https://github.com/user-attachments/assets/50e71e2d-a757-41ec-94d2-b3d5715b1e36">

**Comment by nhan000** [comment]
I agree that it'd be nice to have a filter to show all multilingual voices.

**Comment by gexgd0419** [comment]
This feature has been added in a new release [v0.2.6](https://github.com/gexgd0419/NaturalVoiceSAPIAdapter/releases/tag/v0.2.6).
